docs/samples/v1beta1/torchserve/v1/bloom/README.md

```diff
@@ -0,0 +1,183 @@
+# TorchServe example with Huggingface BLOOM model
+In this example we will show how to serve [Huggingface Transformers with TorchServe](https://github.com/pytorch/serve/tree/master/examples/Huggingface_Transformers)
+on KServe.
+
+## Model archive file creation
+
+Clone [pytorch/serve](https://github.com/pytorch/serve) repository,
+navigate to `examples/Huggingface_Transformers` and follow the steps for creating the MAR file including serialized model and other dependent files.
+TorchServe supports both eager model and torchscript and here we save as the pretrained model. 
+ 
+```bash
+torch-model-archiver --model-name BLOOMSeqClassification --version 1.0 \
+--serialized-file Transformer_model/pytorch_model.bin \
```

Comment:
how is this different from the other BERT model example?

Reply:
This example covers, how to load and run inference for large HF models with Torchserve-KServe.